src/stylesheets/components/_icon.scss

```diff
@@ -0,0 +1,11 @@
+.sm {
+  @include u-height('105');
+}
+
+.md {
+  @include u-height(6);
+}
+
+.lg {
+  @include u-height('10');
+}
```

Comment:
could we use this instead https://fontawesome.com/how-to-use/on-the-web/styling/sizing-icons